modules/Bio/EnsEMBL/Hive/HivePipeline.pm

```diff
@@ -544,151 +544,203 @@ sub print_diagram {
 sub apply_tweaks {
     my $self    = shift @_;
     my $tweaks  = shift @_;
-
+    my @responce;
+    my $responceStructure;
     my $need_write = 0;
-
+    $responceStructure->{Tweaks} = [];
     foreach my $tweak (@$tweaks) {
-        print "\nTweak.Request\t$tweak\n";
+        push @responce, "Tweak.Request  \t$tweak\n";
```

Comment:
Why are you adding a blank space in all those strings ?

Reply:
To make output consistent, all output lines to have the same space after action string